### DISCUSSION TITLE
是不是修好了？

### DISCUSSION THREAD

**Discussion by Naoki19880107** [getlantern/lantern-forum-cn#332] [Q&A]
是不是修好了？我现在7.8.3 mac 反复开关都是好的

**Comment by jean-reno** [comment]
感觉好了  不闪退了 7.8.1 (20240229.143638)

**Comment by MaxMKMax** [comment]
@Naoki19880107 @jean-reno 
我已经在周末和假期连续几天帮你们追他们修了. 应该是修好了吧.
